frontend/src/lib/logic/promptLogic.tsx

```diff
@@ -126,7 +126,7 @@ export function cancellablePrompt(config: Pick<PromptProps, 'title' | 'placehold
             if (unmountResult && div.parentNode) {
                 div.parentNode.removeChild(div)
             }
-            if (value !== undefined) {
```

Comment:
this was trying to resolve even if the payloads were errors/weren't from the input